### PR TITLE
durathread items [fuck github desktop edition]

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
add:durathread backpack
add: durathread toolbelt
add: durathread bandolier
fix: durathread hat doesnt look like a reskinned hardhat
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -27,6 +27,48 @@
 	. = ..()
 	update_icon()
 
+/obj/item/storage/belt/durathread
+	name = "durathread toolbelt"
+	desc = "A toolbelt made out of durathread, it seems resistant enough to hold even big tools like an RCD, it also has higher capacity."
```
Comment (ATH1909) [inline code/game/objects/items/storage/belt.dm]: I feel like that last comma should be replaced by a semicolon or a period.

Comment (fludd12) [reply code/game/objects/items/storage/belt.dm]: Alternative phrasing:
`"A toolbelt made out of durathread. It seems resistant enough to hold larger tools, such as an RCD; it also has a higher capacity."`

**Comment by optimumtact** [comment]
remake this PR but actually document the powercreep with storage and no slowdown on the duffel

**Comment by thebestcodertoevercode** [comment]
we need this pr back but without the "uNdOcUmEnTeD cHaNgEs"

**Pull request closed without merging**